packages/react-native-reanimated/Common/cpp/reanimated/LayoutAnimations/LayoutAnimationsProxy.cpp

```diff
@@ -784,13 +784,12 @@ void LayoutAnimationsProxy::maybeRestoreOpacity(
 void LayoutAnimationsProxy::maybeUpdateWindowDimensions(
     facebook::react::ShadowViewMutation &mutation,
     SurfaceId surfaceId) const {
-  // This is a hacky way to obtain the window dimensions.
-  // We can identify the root, by checking if its tag is equal to the surfaceId
-  if (mutation.parentShadowView.tag == surfaceId) {
+  if (mutation.type == ShadowViewMutation::Update &&
+      !std::strcmp(mutation.oldChildShadowView.componentName, "RootView")) {
```

Comment:
Instead of hard-coding `"RootView"` here, can we use `RootComponentName` from `RootShadowNode.h`?